engines/director/lingo/lingo.cpp

```diff
@@ -46,6 +46,93 @@ namespace Director {
 
 Lingo *g_lingo;
 
+static struct CharNormProto {
```

Comment:
Consider creating a new file for this, like `lingo-utils.cpp` and move this utility function and the table there, so we have better logic separation.

Reply:
Moved to a new file.